zap/src/test/resources/org/zaproxy/zap/spider/parser/html/InputElementsSpiderHtmlParser.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+<base href="http://example.com/sample/">
+<meta charset="UTF-8">
+<title>Input Elements - Spider HTML Parser</title>
+</head>
+<body>
+<!--  Ignored: -->
+<input>
+
+<!--  src -->
```

Comment:
Both comments seem to be preceded by double space.

Reply:
Thought it was a convention, also other comments are with double spaces. But can be fixed for clarity

Reply:
Probably a miss on our part in other files. Not a blocker, but it looks out of place to me.